src/main/java/seedu/data/data.txt

```diff
@@ -0,0 +1,5 @@
+E | 1 | project meeting | Aug 6th 2-4pm
```

Comment:
Should this file be added to `.gitignore`?

Reply:
Second this.